plugins/Login/PasswordResetter.php

```diff
@@ -175,11 +176,12 @@ public function initiatePasswordResetProcess($loginOrEmail, $newPassword)
 
         $login = $user['login'];
 
-        $this->savePasswordResetInfo($login, $newPassword);
+        $keySuffix = time() . Common::getRandomString();
```

Comment:
@diosmosis I reckon the only thing I'd change is to use `Common::getRandomString($len = 32)` just to have even more randomness. Otherwise looks good to me and it worked 👍